<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationDialog</class>
 <widget class="QDialog" name="CalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="99,1">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_38">
       <item>
        <widget class="QSpinBox" name="spinBox_rows">
         <property name="prefix">
          <string>r:</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_colums">
         <property name="prefix">
          <string>c:</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_load">
         <property name="text">
          <string>load</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_calibrations"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_51">
       <item>
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>w:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_w">
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>h:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_h">
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_52">
       <item>
        <widget class="QLabel" name="label_40">
         <property name="text">
          <string>fx:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_fx"/>
       </item>
       <item>
        <widget class="QLabel" name="label_42">
         <property name="text">
          <string>fy:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_fy"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_53">
       <item>
        <widget class="QLabel" name="label_44">
         <property name="text">
          <string>cx:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_cx"/>
       </item>
       <item>
        <widget class="QLabel" name="label_46">
         <property name="text">
          <string>cy:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_cy"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_54">
       <item>
        <widget class="QLabel" name="label_48">
         <property name="text">
          <string>k1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_k1"/>
       </item>
       <item>
        <widget class="QLabel" name="label_50">
         <property name="text">
          <string>p1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_p1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_55">
       <item>
        <widget class="QLabel" name="label_52">
         <property name="text">
          <string>k2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_k2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_54">
         <property name="text">
          <string>p2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_p2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_56">
       <item>
        <widget class="QLabel" name="label_56">
         <property name="text">
          <string>k3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_k3"/>
       </item>
       <item>
        <widget class="QLabel" name="label_65">
         <property name="text">
          <string>err:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_err"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_calibrate">
         <property name="text">
          <string>calibrate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_applyCalibration">
         <property name="text">
          <string>apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
